### ISSUE TITLE
the effect of Squeezing Budget 

### ISSUE THREAD

**Issue by self-reward** [VILA-Lab/SRe2L#19]
Dear Author,

Thank you for your excellent work! I have some detailed questions about your paper that I hope you can clarify.

1 In Table 2, the results on Tiny-ImageNet suggest an increasing challenge in data recovery from a model that has been trained over more iterations. When distilling knowledge from ImageNet-1K, could I achieve better performance with a ResNet trained a little iterations from scratch on Tiny-ImageNet rather than using a pre-trained model?

2 In Table 5, all the evaluation models contain Batchnorm layers, but I am curious whether it is possible to test using networks without Batchnorm?

I look forward to your response.

**Comment by zeyuanyin** [comment]
Thanks for your interest.

1. The phenomenon of an increasing challenge when recovering from a model trained with more iterations is validated in CIFAR and Tiny-ImageNet. Thus, I think it should have existed in the ImageNet-1K dataset. In the paper, we use the pre-trained ResNet models in order to save GPU resources and time. Kindly note that the pre-trained ResNet is trained with 90 epochs using the scripts at https://github.com/pytorch/vision/tree/main/references/classification. 

2. Yes, you can find the results of evaluation models without Batchnorm in Table 20 of CDA paper (https://arxiv.org/abs/2311.18838v1). We present the related results here.

    | Model        | AlexNet  | DeiT-Base  | VGG-11  | VGG-13  | VGG-16  | VGG-19 |
    |--------------|---------:|-----------:|--------:|--------:|--------:|-------:|
    | accuracy (%) |   14.60  |     30.27  |  36.99  |  38.60  |  42.28  |  43.30 |

**Comment by self-reward** [comment]
Thank you for your response, it helped me a lot.
